app/config/config.yml

```diff
@@ -5,7 +5,7 @@ imports:
     - { resource: global_view_parameters.yml }
 
 parameters:
-    locales: [en, nl]
+    locales: [en, nl, pt]
```

Comment:
Is this OK with you @thijskh? Adding PT to the list of locales will show the Portuguese language in the language switcher. Overriding the locales from the `parameters.yml` does not work. 

Two options:
1. Leave it as is, and we include this feature for our own Portuguese speaking users
2. Move the locale parameter definition to the parameters.yml and remove it from the config.yml, allowing configuration of the locale switcher from the parameters.yml.

Reply:
Thank you guys.
When I see some spelling or grammar in Portuguese that is not in the best way I will correct and submit the changes.

Reply:
2

Reply:
Then let's merge this PR and I will make the config configurable in a new PR